Blocks/HTTP/test.cc

```diff
@@ -427,6 +436,10 @@ TEST(HTTPAPI, PostCerealizableObject) {
   });
   EXPECT_EQ("Data: {\"data\":{\"x\":42,\"s\":\"foo\"}}",
             HTTP(POST(Printf("http://localhost:%d/post", FLAGS_net_api_test_port), SerializableObject())).body);
+
+  EXPECT_EQ(
+      "Data: {\"custom_name\":{\"y\":100}}",
```

Comment:
By design `NamedSerializableObject` has `JSONEntryName() { return "custom_name"; }`.

So that we can finally have proper top-level JSON response object names when we want to -- like I just did yesterday with the profiler.